.github/integrationBenchScripts/07_execBench.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#set -eo pipefail
+
+k=$1
+GITHUB_RUN_ID=$2
+PROVER=$3
+
+export GOROOT="/usr/local/go"
+export GOPATH="$HOME/go"
+export PATH="$GOROOT/bin:$GOPATH/bin:$PATH"
+
+# Get the latest temp directory in the home directory
+current_dir="$HOME"/CI_Prover_Benches/"$GITHUB_RUN_ID"
+
+target_dir="$current_dir/zkevm-circuits"
+
+printf -v _date '%(%Y-%m-%d_%H:%M:%S)T' -1
```

Comment:
Does this format data in UTC or local machine date?

Reply:
it is set to UTC by
./weeklyBenchScripts/02_setup.sh:42:sudo timedatectl set-timezone UTC
which is called by
./integrationBenchScripts/cloud-tests-local-trigger.sh:23:  ssh -i ~/.ssh/bench.pem -o StrictHostKeyChecking=no ubuntu@"$PROVER_IP" "bash -s" -- <../weeklyBenchScripts/02_setup.sh